src/components/MediaTile/MediaTile.tsx

```diff
@@ -39,63 +41,67 @@ const useStyles = makeStyles(
       top: 0,
       width: 148
     },
-    imageOverlayShow: {
-      "&$imageOverlay": {
+    mediaOverlayShadow: {
+      "&mediaOverlay": {
         alignItems: "center",
         display: "flex",
         justifyContent: "center"
       }
     },
-    imageOverlayToolbar: {
+    mediaOverlayToolbar: {
       display: "flex",
       justifyContent: "flex-end"
     }
   }),
-  { name: "ImageTile" }
+  { name: "MediaTile" }
 );
 
-interface ImageTileProps {
-  image: {
-    alt?: string;
+interface MediaTileProps {
+  media: {
+    alt: string;
     url: string;
+    type?: string;
   };
   loading?: boolean;
-  onImageDelete?: () => void;
-  onImageEdit?: (event: React.ChangeEvent<any>) => void;
+  onDelete?: () => void;
+  onEdit?: (event: React.ChangeEvent<any>) => void;
```

Comment:
Is there a point of having it optional?

Reply:
Yes, MediaTile is being used in several places where for some reason we want to only delete the image instead of editing it, eg. CategoryBackground.